### DISCUSSION TITLE
Bulk Share Snippets

### DISCUSSION THREAD

**Discussion by jtmuller5** [pieces-app/support#392] [Ideas]
Hi,

I am a new pieces user and I was wondering if there's a way to share multiple snippets at once.

I would like to create a "package" of snippets that people can use for building Flutter apps but it seems like they would need to save each snippet separately.

It would be ideal if I could share a bundle of snippets that would stay updated as I make improvements to them.

Thanks!

**Comment by rosie-at-pieces** [comment]
Great idea @jtmuller5 - we've heard this from some other users too!

Link to internal ticket [#987](https://github.com/pieces-app/pieces_for_x/issues/987)
